cost/aws/reserved_instances/recommendations/aws_reserved_instance_recommendations.pt

```diff
@@ -507,9 +517,6 @@ policy "aws_ri_recommendations" do
       field "databaseEdition" do
         label "Database Edition"
       end
-      field "databaseEngine" do
```

Comment:
Why are some fields deleted like this?

Reply:
I was told that I should map DB Engine to Platform. So if Platform already has the database engine, I didn't see the use of this field. Also, excel file says that databaseEngine should be renamed to Platform.

Reply:
Both fields should be present, as far as I know. The platform field is more extensive and includes more information, but the databaseEngine field is still the default field in the scraper service and should be used

Reply:
Got it. I've added it again